packages/client/hooks/useControlBarCovers.ts

```diff
@@ -42,6 +42,7 @@ export const useCoverable = (
   parentRef?: RefObject<HTMLDivElement>,
   columnsRef?: RefObject<HTMLDivElement>
 ) => {
+  if (id === 'poker-thread') return true
```

Comment:
hooks must always run in the same order every single time.
if we conditionally return early, that isn't true (i mean, with our current business logic it is because you can't get from 1 meeting type to another without unmounting this component, but as a best practice, we should follow the rules of hooks).


 

the magic string feels a little brittle (when i call this hook, i don't know that the string has to match a certain template).
i'd suggest the hook takes a `meetingType` variable to be a little more explicit.